### PR TITLE
feat: add Webhooks endpoints

### PR DESCRIPTION
**Pull request by kvithana**
Added Webhooks endpoints as per #3. With this PR we've covered all the current actions currently supported by the Up API!

### CODE REVIEW

File: src/webhooks/api.ts [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  }
+
+  /**
+   * Delete a specific webhook by providing its unique identifier. Once deleted,
+   * webhook events will no longer be sent to the configured URL.
+   * @param id The unique identifier for the webhook. e.g.
+   * a3f1e92b-b790-42cf-afe7-6f4efad9fa9d
+   */
+  public async delete(id: string): Promise<void> {
+    return this.api.delete(`${ENDPOINT}/${id}`);
```
Comment (ndench) [inline src/webhooks/api.ts:79]: I'm unsure if this is necessary to make typescript happy and get proper IDE completion?

```suggestion
    return this.api.delete<void, void>(`${ENDPOINT}/${id}`);
```

**Review by ndench — APPROVED** [review]
Thanks @kvithana! It's awesome to have full compatibility with the Up API!

I just left one comment that I'm not sure about, let me know what you think.

File: src/webhooks/api.ts (hunk 1 above)
Comment (kvithana) [reply src/webhooks/api.ts:79]: I believe that if you leave it unspecified it assumes void as the type. There are no complaints on my IDE and it seems it's assumed <void,void> when I hover over it to inspect.

Comment (ndench) [reply src/webhooks/api.ts:79]: Ok awesome!

**Pull request merged**